### ISSUE TITLE
Error when running 'gridsome develop' on newly created project

### ISSUE THREAD

**Issue by jscheffner** [gridsome/gridsome#187]
When running `gridsome develop` in the directory I created using `gridsome create hello-gridsome`, the following error occurs:

````
C:\Users\Me\Workspace\hello-gridsome\node_modules\ajv\lib\keyword.js:65
      throw new Error('custom keyword definition is invalid: '  + this.errorsText(validateDefinition.errors));
      ^

Error: custom keyword definition is invalid: data/errors should be boolean
    at Ajv.addKeyword (C:\Users\Me\Workspace\hello-gridsome\node_modules\ajv\lib\keyword.js:65:13)
    at module.exports (C:\Users\Me\Workspace\hello-gridsome\node_modules\ajv-errors\index.js:10:7)
    at Object.<anonymous> (C:\Users\Me\Workspace\hello-gridsome\node_modules\webpack\node_modules\schema-utils\src\validateOptions.js:22:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\Me\Workspace\hello-gridsome\node_modules\webpack\node_modules\schema-utils\src\index.js:7:25)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
````

### Steps to reproduce

````
> gridsome create hello-gridsome
> cd hello-gridsome
> gridsome develop
````

### Environment:

- Windows
- Node v10.15.1
- gridsome cli v0.0.7

**Comment by blubMe** [comment]
its same , the problem is from ajv dependency

**Comment by blubMe** [comment]
Quick fix its just add this code in the package.json

```json
  "resolutions": {
    "ajv": "6.8.1"
  }
```

**Comment by jscheffner** [comment]
@blubMe: Thanks, that did the trick.

**Comment by jscheffner** [comment]
Got fixed with ajv [v6.9.1](https://github.com/epoberezkin/ajv/issues/941#issuecomment-462114537).
